specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/entityTypes/IntegrationRuntime.json

```diff
@@ -0,0 +1,1537 @@
+      ]
+    },
+    "LicensedComponentSetupTypeProperties": {
+      "description": "Installation of licensed component setup type properties.",
+      "type": "object",
+      "properties": {
+        "componentName": {
+          "description": "The name of the 3rd party component.",
+          "type": "string"
+        },
+        "licenseKey": {
+          "description": "The license key to activate the component.",
+          "$ref": "../datafactory.json#/definitions/SecretBase"
+        }
+      },
+      "required": [
+        "componentName"
+      ]
+    },
+    "AzPowerShellSetup": {
+      "description": "The express custom setup of installing Azure PowerShell.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/CustomSetupBase"
+        }
+      ],
+      "properties": {
+        "typeProperties": {
+          "description": "Install Azure PowerShell type properties.",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/AzPowerShellSetupTypeProperties"
+        }
+      },
+      "required": [
+        "typeProperties"
+      ]
+    },
+    "AzPowerShellSetupTypeProperties": {
+      "description": "Installation of Azure PowerShell type properties.",
+      "type": "object",
+      "properties": {
+        "version": {
+          "description": "The required version of Azure PowerShell to install.",
+          "type": "string"
+        }
+      },
+      "required": [
+        "version"
+      ]
+    },
+    "EntityReference": {
+      "description": "The entity reference.",
+      "type": "object",
+      "properties": {
+        "type": {
+          "description": "The type of this referenced entity.",
+          "type": "string",
+          "enum": [
+            "IntegrationRuntimeReference",
+            "LinkedServiceReference"
+          ],
+          "x-ms-enum": {
+            "name": "IntegrationRuntimeEntityReferenceType",
+            "modelAsString": true
+          }
+        },
+        "referenceName": {
+          "description": "The name of this referenced entity.",
+          "type": "string"
+        }
+      }
+    },
+    "SelfHostedIntegrationRuntime": {
+      "x-ms-discriminator-value": "SelfHosted",
+      "description": "Self-hosted integration runtime.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/IntegrationRuntime"
+        }
+      ],
+      "properties": {
+        "typeProperties": {
+          "description": "When this property is not null, means this is a linked integration runtime. The property is used to access original integration runtime.",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/SelfHostedIntegrationRuntimeTypeProperties"
+        }
+      }
+    },
+    "SelfHostedIntegrationRuntimeTypeProperties": {
+      "description": "The self-hosted integration runtime properties.",
+      "type": "object",
+      "properties": {
+        "linkedInfo": {
```

Comment:
same here, this property also needs to be removed Linked IR is not supported in Synapse